rules-tests/CodeQuality/Rector/Isset_/IssetOnPropertyObjectToPropertyExistsRector/Fixture/collection.php.inc

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Rector\Tests\CodeQuality\Rector\Isset_\IssetOnPropertyObjectToPropertyExistsRector\Fixture;
+
+/**
+ * @template TKey of array-key
+ *
+ * @template-covariant TValue
+ *
+ * @implements \ArrayAccess<TKey, TValue>
+ */
+class Collection implements ArrayAccess, CanBeEscapedWhenCastToString, Enumerable {}
```

Comment:
it become ArrayAccess, CanBeEscapedWhenCastToString, and Enumerable inside namespace, add `\` prefix if you don't want to add to use statement

```suggestion
class Collection implements \ArrayAccess, \CanBeEscapedWhenCastToString, \Enumerable {}
```